tensorboard/webapp/alert/views/alert_display_snackbar_container.scss

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+:host {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.message {
+  font-size: 14px;
```

Comment:
The font-size on the controls is also 14px, so I'd prefer keeping it consistent.

Reply:
Ah true, it is not explicit.  I showed the screenshots to our UX designer, and for now the default button font size is reasonable.

Reply:
My point was that it should probably be set explicitly instead of using the default.

Reply:
Per offline discussion:

It might be strange if a message body was way smaller than the button text.  However, it is not a goal in this view to constrain those 2 font sizes to be equal.